### PR TITLE
Task #211600 - [WEB-UI] - To add prerak id on menu

### PR DESCRIPTION
**Pull request by sonaliTekdi**
### I have ensured that following `Pull Request Checklist` is taken care of before sending this PR
* [ ] Code is formatted as per format decided
* [ ] Updated acceptance criteria in tracker
* [ ] Updated test cases in test-cases-tracker
Link - https://tracker.tekdi.net/issues/211600
Screenshot from 2024-01-10 17-24-38

## Summary by CodeRabbit

- **New Features**
	- The app's AppBar now displays a user-specific ID retrieved from local storage.

### CODE REVIEW

File: lib/common-lib/src/components/layout/AppBar.js [hunk 1]
```diff
@@ -60,6 +59,12 @@ export default function AppBar({
   const [isOpen, setIsOpen] = React.useState(false)
   const [progress, setProgress] = useState(0)
   const [status, setStatus] = React.useState('')
+  const [prerakID, setPrerakId] = React.useState()
```
[the remark this answers is not in the document]
Comment (sonaliTekdi) [reply lib/common-lib/src/components/layout/AppBar.js:62]: done

File: lib/common-lib/src/components/layout/AppBar.js [hunk 2]
```diff
@@ -60,6 +59,12 @@ export default function AppBar({
...
+
+  useEffect(() => {
+    const prerakId = localStorage.getItem('id')
+    setPrerakId(prerakId)
+  }, [])
```
[the remark this answers is not in the document]
Comment (sonaliTekdi) [reply lib/common-lib/src/components/layout/AppBar.js:67]: done, handled error using try and catch

File: lib/common-lib/src/components/layout/AppBar.js [hunk 3]
```diff
@@ -443,6 +452,9 @@ export default function AppBar({
                               facilitator ? facilitator?.status : 'under_review'
                             }
                           />
+                          <FrontEndTypo.H3 pt='15px'>
+                            ID: {prerakID}
```
Comment (sagarkoshti1990) [inline lib/common-lib/src/components/layout/AppBar.js:456]: ID should be language-contant

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

File: lib/common-lib/src/components/layout/AppBar.js (hunk 3 above)
Comment (sagarkoshti1990) [reply lib/common-lib/src/components/layout/AppBar.js:456]: also add to profile page.

**Review by sagarkoshti1990 — CHANGES_REQUESTED** [review]

**Review by sagarkoshti1990 — APPROVED** [review]

**Pull request merged**